core/setup.py

```diff
@@ -64,30 +64,28 @@
         # ----
         # These packages are major-version-0. Keep upper bounds on upcoming minor versions (which could have breaking changes)
         # and check compatibility / bump in each new minor version of dbt-core.
-        "colorama>=0.3.9,<0.5",
         "pathspec>=0.9,<0.12",
-        "isodate>=0.6,<0.7",
-        # ----
         "sqlparse>=0.2.3,<0.5",
         # ----
         # These are major-version-0 packages also maintained by dbt-labs. Accept patches.
         "dbt-extractor~=0.5.0",
         "minimal-snowplow-tracker~=0.0.2",
         "dbt-semantic-interfaces~=0.5.0a2",
+        "dbt-common @ git+https://github.com/dbt-labs/dbt-common.git@copy-code-from-dbt-core#egg=dbt",
         # ----
         # Expect compatibility with all new versions of these packages, so lower bounds only.
-        "jsonschema>=3.0",
+        # "jsonschema>=3.0", # TODO: looks like we only use machumaro.jsonschema in core now?
         "packaging>20.9",
         "protobuf>=4.0.0",
         "pytz>=2015.7",
         "pyyaml>=6.0",
         "typing-extensions>=4.4",
         # ----
         # Match snowflake-connector-python, to ensure compatibility in dbt-snowflake
-        "cffi>=1.9,<2.0.0",
-        "idna>=2.5,<4",
-        "requests<3.0.0",
-        "urllib3~=1.0",
+        # "cffi>=1.9,<2.0.0",  TODO: move these to adapters since they're for snowflake compatibility?
+        # "idna>=2.5,<4",
+        # "requests<3.0.0",
+        # "urllib3~=1.0",  TODO: also referenced in logger.py.
```

Comment:
Open question on if these should move to dbt-adapter since they're dbt-snowflake specific.

Reply:
Resolved on slack: yes!